format/src/main/thrift/carbondata.thrift

```diff
@@ -183,11 +183,23 @@ struct FileFooter{
 }
 
 /**
+* Footer for indexed carbon file
+*/
+struct FileFooterVersion3{
```

Comment:
name it as `FileFooter3` like others, `BlockletInfo3` indicate it is BlockletInfo version 3.